src/constant/lendingPairs.ts

```diff
@@ -15,8 +13,6 @@ export const lendingPairs: LendingPairsConfig = [
   {
     chainId: arbitrum.id,
     pairAddress: "0xB900A00418bbD1A1b7e1b00A960A22EA540918a2",
```

Comment:
should we put this address into the addresses file (addresses.ts) or even better with chainConfig?

Reply:
We should discuss this in more detail, but the reason for this not being part of the existing chainConfig is to enable support for the lending pairs regardless of which chain the user is connected to. If we move chainConfig to a format like this (with `chainId` defined within it instead of having separate per-chain objects), we can merge the files.